README.md

```diff
@@ -39,8 +39,16 @@ ln -fs openshift-ansible-*<version>* openshift-ansible
 
 | openshift-ansible url     | OpenShift version | 
 |:-------------------------:|:-----------------:|
+| [3.5.77-1](https://github.com/openshift/openshift-ansible/archive/openshift-ansible-3.5.77-1.tar.gz) | OCP 3.5.x |
 | [3.4.50-1](https://github.com/openshift/openshift-ansible/archive/openshift-ansible-3.4.60-1.tar.gz) | OCP 3.4 |
 
+* Run `ansible-galaxy` to pull in the necssary requirements for the CASL provisioning:
```

Comment:
@oybed prior to this change, if the user was using our container image to run this installs, there was no requirement to actually have ansible installed on localhost. With ansible-galaxy, it looks like we are adding a requirement for ansible. Is that true, and are we ok with that? If so, we need to document in the README, that the user needs to install `ansible` and which versions we support.